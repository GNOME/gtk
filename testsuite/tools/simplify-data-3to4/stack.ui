<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <!-- interface-requires gtk+ 3.0 -->
  <object class="GtkWindow" id="window1">
    <property name="can_focus">False</property>
    <property name="type">popup</property>
    <child>
      <object class="GtkBox" id="box1">
        <property name="visible">True</property>
        <child>
          <object class="GtkStackSwitcher" id="stackswitcher1">
            <property name="visible">True</property>
            <property name="stack">stack1</property>
          </object>
        </child>
        <child>
          <object class="GtkStack" id="stack1">
            <property name="visible">True</property>
            <child>
              <object class="GtkButton" id="page1">
                <property name="label" translatable="yes">Yes</property>
                <property name="visible">True</property>
              </object>
              <packing>
                <property name="name">page1</property>
                <property name="title">Page 1</property>
                <property name="needs-attention">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="page2">
                <property name="label" translatable="yes">No</property>
                <property name="visible">True</property>
              </object>
              <packing>
                <property name="name">page2</property>
                <property name="title">Page 2</property>
                <property name="icon-name">page2-icon</property>
              </packing>
            </child>
          </object>
        </child>
      </object>
    </child>
  </object>
</interface>
