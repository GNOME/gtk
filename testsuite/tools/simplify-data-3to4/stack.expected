<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkWindow" id="window1">
    <property name="type">popup</property>
    <child>
      <object class="GtkBox" id="box1">
        <child>
          <object class="GtkStackSwitcher" id="stackswitcher1">
            <property name="stack">stack1</property>
          </object>
        </child>
        <child>
          <object class="GtkStack" id="stack1">
            <child>
              <object class="GtkStackPage">
                <property name="name">page1</property>
                <property name="title">Page 1</property>
                <property name="needs-attention">1</property>
                <property name="child">
                  <object class="GtkButton" id="page1">
                    <property name="label" translatable="yes">Yes</property>
                  </object>
                </property>
              </object>
            </child>
            <child>
              <object class="GtkStackPage">
                <property name="name">page2</property>
                <property name="title">Page 2</property>
                <property name="icon-name">page2-icon</property>
                <property name="child">
                  <object class="GtkButton" id="page2">
                    <property name="label" translatable="yes">No</property>
                  </object>
                </property>
              </object>
            </child>
          </object>
        </child>
      </object>
    </child>
  </object>
</interface>
